src/trigger/text_area.cpp

```diff
@@ -33,7 +38,8 @@ TextArea::TextArea(const ReaderMapping& mapping) :
   m_fade_delay(1.0f),
   m_current_text(0),
   m_status(Status::NOT_STARTED),
-  m_timer()
+  m_timer(),
+  m_anchor(0)
 {
```

Comment:
It should now default to the old default option, `ANCHOR_MIDDLE`.

Reply:
I was trying to do like this but AnchorPoint uses bitmasks so it is not ordered enum